my-app/src/components/LoginPanel/Login.js

```diff
@@ -22,36 +24,37 @@ class Login extends Component {
 
   login(e) {
     e.preventDefault();
-    fire.auth().signInWithEmailAndPassword(this.state.email, this.state.password).then((u)=>{
+    fire.auth().signInWithEmailAndPassword(this.state.email, this.state.password).then((u) => {
     }).catch((error) => {
-        console.log(error);
-      });
+      alert("Błędne hasło lub nazwa użytkownika");
```

Comment:
I was thinking about some more fancy way - like some toast/notification, but lets keep it that way :D